lib/mwoffliner.lib.js

```diff
@@ -623,6 +625,17 @@ module.exports = {
                 });
         }
 
+        function saveStaticKiwixFiles( finished ) {
+            try {
+                fs.readFile('./kiwix_mobile.css', (err, data) =>
```

Comment:
Please remove "kiwix" from the naming (for the file too), just call it "mobile.css".